### ISSUE TITLE
Proposal: include Aurora Services

### ISSUE THREAD

**Issue by wmww** [lineageos4microg/android_prebuilts_prebuiltapks#50]
TLDR: I propose you include `Aurora Services` as a system app

Of the three methods for installing Play Store apps mentioned on the website (PlayMaker, GPlayWeb and Yalp Store), none of them appear to be actively maintained and only Yalp Store is usable by those unwilling or unable to set up their own server.

[Aurora Store](https://f-droid.org/en/packages/com.aurora.store/) seems to be a (the most?) accessible and well maintained Play store client. It can install apps using root, by prompting the user or with [Aurora Services](https://gitlab.com/AuroraOSS/AuroraServices) if that is installed as a system app.

Sideloading system apps and keeping them up to date seems to be difficult/impossible if you don't use Magisk (in my case I don't want to deal with the added complexity of OTAs that Magisk incurs). If Lineage for microG included Aurora Services by default, any user who installed Aurora would get auto app installations without the hassle. The value proposition is similar to that of the F-Droid Privileged Extension you already include (which is awesome).

Thanks for all your hard work, please let me know if this isn't the appropriate place to make such proposals.

EDIT: if the idea is accepted and it's just a matter of infrastructure work to make it happen, I'm more than happy to do that if pointed in the right direction.

**Comment by bananer** [comment]
With Android 12 Aurora Services should not be needed anymore: https://www.xda-developers.com/android-12-alternative-app-stores-update-apps-background/

I would like to keep the scope of the project to LineageOS and microG.
